test/extended/oauth/expiration.go

```diff
@@ -114,7 +114,8 @@ func testTokenFlow(oc *util.CLI, newRequestTokenOptions oauthserver.NewRequestTo
 }
 
 func testCodeFlow(oc *util.CLI, newRequestTokenOptions oauthserver.NewRequestTokenOptionsFunc, client *oauthv1.OAuthClient, expectedExpiresIn int32) {
-	anonymousClientConfig := rest.AnonymousClientConfig(oc.AdminConfig())
+	anonymousClientConfig, err := oauthserver.InjectRouterCA(oc, rest.AnonymousClientConfig(oc.AdminConfig()))
```

Comment:
`oc.AdminConfig()` should already have all the certs required

Reply:
seems like my kubeconfig itself does not have the certificate, as you mentioned I need to go back and check installer is not injecting the router CA!